components/typography/index.css

```diff
@@ -211,15 +211,55 @@ governing permissions and limitations under the License.
 /* t-shirt size-based typography */
 
 /* Headings t-shirt sizes */
+/* Standard SUIT rules for Heading: regular, serif, heavy, light */
 .spectrum-Heading {
   font-family: var(--spectrum-heading-xl-text-font-family);
+  font-weight: var(--spectrum-heading-xl-text-font-weight);
+
+  em, .spectrum-Heading--emphasis {
+    font-style: var(--spectrum-heading-xl-emphasis-text-font-style);
+    font-weight: var(--spectrum-heading-xl-emphasis-text-font-weight);
+  }
+
+  strong, .spectrum-Heading--strong {
```

Comment:
This selector targets:
1. child elements of `spectrum-Heading` which are the `strong` tag
2. child elements of `spectrum-Heading` which have the classname `spectrum-Heading--strong`

1 is correct, but 2 is wrong. `spectrum-Heading--strong` is a modifier that should be applied to the outer element, `spectrum-Heading`, such that it affects everything inside of it, right? So it should be:

```
strong, &.spectrum-Heading--strong {
```

Right? That is the intent here, unless I'm missing something? Same should apply to the other typography components.

Reply:
That is not the intend here. 
Elements nested inside `.spectrum-Heading` used to be only strong or em tags. 
@NateBaldwinDesign  requested to have those also be applied to all all kinds of other elements ergo the `.spectrum-Heading--strong` class. 
By adding the `&`on the .spectrum-Heading--strong class we are applying the selector in combination with the parent selector.
The intend here is to have elements with the classname `.spectrum-heading--strong` inside a `spectrum-Heading` container have the same style like the strong tag.

Reply:
In that case, that's the wrong naming convention. You're defining it as a sub-element, so it should use a single dash, i.e. `spectrum-Heading-strong`. In addition, it does not need to be nested inside of `spectrum-Heading`, that adds unnecessary specificity.

Reply:
I have reverted the naming and removed the double dashed. 

I still suggest that we keep this specificity as it is unclear to me how this change would impact for everyone upgrading to Typography V4. 
Again, the intent of this PR is to implement Typography V4 from a CSS perspective without impacting DOM structure or requiring DOM rewrite.s